src/helpers/parser/sueParser.js

```diff
@@ -28,21 +28,21 @@ export const parseSueData = (data, appDesignSystem) => {
 
     return {
       ...item,
-      address: item.address.replace('\r\n ', '\r\n'),
+      address: item.address?.replace('\r\n ', '\r\n'),
       aspectRatio: parsedImageAspectRatio('361:203'),
       bottomDivider: false,
-      iconName: matchedStatus.iconName,
+      iconName: matchedStatus?.iconName,
       params: {
-        title: item.title,
+        title: item?.title,
         query: QUERY_TYPES.SUE.REQUESTS_WITH_SERVICE_REQUEST_ID,
-        queryVariables: { id: item.serviceRequestId },
+        queryVariables: { id: item?.serviceRequestId },
         rootRouteName: ROOT_ROUTE_NAMES.SUE,
         bookmarkable: false,
         details: item
       },
       picture: { url: mainImageOfMediaContents(parsedMediaUrl) },
-      routeName: ScreenName.Detail,
-      status: matchedStatus.status,
+      routeName: ScreenName?.Detail,
```

Comment:
a ScreenName should be always present, we do not need to check it

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/966/commits/2b603bd8f6f8b43bf449efd90fce208da3b2c23c